### ISSUE TITLE
Stuck on loading pyv8 binary, please wait

### ISSUE THREAD

**Issue by HerrEisenheim** [sergeche/emmet-sublime#704]
I just updated ST Dev from build 3175 to 3179 and suddenly I am seeing pyv8 binary loading is taking forever.

And I am not sure if this is related to emmet but I also have `save_on_focus_lost: true` enabled in my settings but that's not working either after I updated ST3.

**Comment by HerrEisenheim** [comment]
It's a similar issue to #703

**Comment by HerrEisenheim** [comment]
And yes, `save_on_focus_lost: true` issue IS in fact related to emmet. How do I know? When I disable emmet it starts to work again.

**Comment by thevasya** [comment]
Yes, it's loading forever in builds 3179 and 3180 for me as well (Ubuntu 64-bit).

**Comment by bdjurkan** [comment]
It's loading forever in builds 3179 and 3180 for me too. Debian stretch x64.

**Comment by dufferzafar** [comment]
I'm seeing this too on build 3180. Switching back to 3176 for now.

**Comment by OnlyDevelops** [comment]
It's not necessarily stuck loading forever for me, it just loads extremely slow every time I start Sublime Text which kind of ruins the point of Sublime Text for me and makes this addon pointless for me to use. I use Sublime Text for it to be fast, and sadly, with this, it is not fast.

**Comment by wowcut** [comment]
Is there a workaround for this? Any help fro the sublime develpers? Emmet is such a basic tool, this is something that should work out of the box in a pro editor like sublime.

**Comment by lingo** [comment]
I can't get it to work after install.  Downloading the pyv8 binaries locks forever.  If I manually put them in place, it still tries to connect to github and locks forever.
If I prevent this from happening by commenting the code, then I can see that `check_context` never finds a valid javascript context, and following this back, I find there's a (swallowed) exception on import of the module:

`
ImportError: ~/user/.config/sublime-text-3/Packages/PyV8/pyv8-linux64-p3/_PyV8.so: undefined symbol: PyExc_ValueError'
`

**Comment by jfcherng** [comment]
I am afraid that this is because the pre-compiled PyV8 module is no longer compatible with the latest ST.

![image](https://user-images.githubusercontent.com/6594915/51901134-ee15b000-23f1-11e9-9811-a05993a1eab9.png)

**Comment by wbond** [comment]
@sergeche Are the build scripts for PyV8 public somewhere? What compilers are you using for the various platforms?

**Comment by sergeche** [comment]
@wbond Don‘t know, actually. The original PyV8 bundle was made by @sublimator 
Also, I have upcoming new Emmet 2 code base, which is smaller and faster and we may not need a heavy PyV8. We can discuss it somewhere in chat

**Comment by Orlmente** [comment]
@sergeche is there any ETA for the upcoming V2?

**Comment by sergeche** [comment]
Sorry, no ETA yet, too busy on my job

**Comment by jfcherng** [comment]
Just a FYI, this issue may have been solved since Sublime Text 3192.

![image](https://user-images.githubusercontent.com/6594915/53720818-9c52c080-3e9c-11e9-9224-71335b15cc30.png)

**Comment by bdjurkan** [comment]
Emmet works on Debian stretch x64 and Win10 with build 3194, 3195.
